package.json

```diff
@@ -25,9 +25,9 @@
     "prettier": "prettier -o --write './**/*.{js,css}'",
     "sass-lint": "sass-lint -v -q hq/public/stylesheets/*.css",
     "not-master": "test \"$BRANCH_NAME\" != \"master\" && echo \"BRANCH_NAME variable is '$BRANCH_NAME', not master\" ",
-    "snyk-test": "PATH=$(pwd):$PATH snyk test --debug --org=guardian --json --file=build.sbt ",
+    "snyk-test": "PATH=$(pwd):$PATH snyk test --debug --org=guardian-cuu --json --file=build.sbt ",
     "snyk-test-master": "npm run not-master --silent || npm run snyk-test --silent ",
-    "snyk-monitor": "PATH=$(pwd):$PATH snyk monitor --debug --org=guardian --file=build.sbt ",
+    "snyk-monitor": "PATH=$(pwd):$PATH snyk monitor --debug --org=guardian-cuu --file=build.sbt ",
```

Comment:
Why `--org=guardian-cuu`? That is not the org that Security HQ sits in...

We probably want `--org=guardian-security` 😄

Reply:
+1, thanks Kate. When I saw it the first time I thought that "cuu" had a secret meaning I didn't know about :)

Reply:
Is it ok to hard code that in the repo?

Reply:
My mistake, I thought that was correct.  I'll fix.